.github/workflows/update-gradle-wrapper.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Set up Zulu JDK 11
         uses: actions/setup-java@v3
         with:
-          distribution: 'zulu'
+          distribution: 'temurin'
```

Comment:
The 'name' also needs to be changed.

<img width="217" alt="image" src="https://user-images.githubusercontent.com/29346818/228431802-99a7b1a3-2d3a-4185-8826-3f5f277c80a0.png">

Reply:
Thank you. I updated it